lib/phoenix/digester/gzip.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Phoenix.Digester.Gzip do
+  @moduledoc ~S"""
+  Gzip compressor for Phoenix.Digester
+  """
+  @behaviour Phoenix.Digester.Compressor
+
+  def compress_file(file_path, content) do
+    if Path.extname(file_path) in Application.fetch_env!(:phoenix, :gzippable_exts) do
+      {:ok, :zlib.gzip(content)}
```

Comment:
Should we also check the file is smaller, like we do for brotli? In any case, that can be done in a separate PR. :)